scenario-metadata.yml

```diff
@@ -0,0 +1,89 @@
+scenarios:
+  - id: scenario01
+    name: "Single Sign On with delegated access control"
+    tests:
```

Comment:
please add test-location property since we changed the structure and to make TestGrid aware on the change.

tests:
      - test-type: jmeter
        test-location: jmeter